src/main/java/com/unblu/usonarstatus/service/Service.java

```diff
@@ -47,6 +47,9 @@ public class Service {
 	@ConfigProperty(name = "gitlab.external.check.name", defaultValue = "SONAR")
 	String externalStatusCheckName;
 
+	@ConfigProperty(name = "branch.bypass", defaultValue = "^mr(\\d+)_.+")
+	String bypassBranchPattern;
```

Comment:
Please change the type to `Optional<String>` to make the config optional. 
And remove the default value.

To have the value set during the test, you can set it in `src/test/resources/application.properties`